src/engine/V3Eval.v3

```diff
@@ -441,108 +498,259 @@ component V3Eval {
 			_ => return val;
 		}
 	}
-}
+	// Private (i.e. file-scoped) utilities.
```

Comment:
Since these are outside the component, they don't need to be indented another level.

Reply:
Oh, I got it. Didn't notice they were moved outside of the component. Pushed a new change.